docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -35,15 +57,25 @@ spec:
       - 'hello world'
 ```
 
-## Step 2: Create a KeptnTask for each run
+This example uses the `container-runtime` runner,
+but you can instead use the `deno-runtime` or `python-runtime` runner.
```

Comment:
can we add links to the descriptions of container-runtime, deno-runtime and python-runtime runners? So the users can click on it and read what the runners actually do

Reply:
done